server/src/main/java/org/spine3/server/entity/Repository.java

```diff
@@ -294,4 +307,44 @@ public int getIndex() {
             return result;
         }
     }
+
+    /**
+     * An iterator of all entities from the storage.
```

Comment:
As this `Iterator` implementation does not support element removal, let's mention that in docs.